themes/mongodb/src/js/componentAccordion.js

```diff
@@ -11,6 +13,12 @@ function accordionShowHide(element) {
 
     const control = element.querySelector('.accordion__action');
     control.innerHTML = (control.innerHTML === 'Expand') ? 'Collapse' : 'Expand';
+
+    const title = element.querySelector('.accordion__title').innerText;
+    reportAnalytics('Accordion Toggled', {
+        'title': title,
+        'status': 'expanded'
```

Comment:
Wouldn't this always have a `status` of `expanded`? This should change based on the state of the accordion.